### PR TITLE
replace deprecated logger methods

### PR DESCRIPTION
**Pull request by juev**
#### Overview

Just replace deprecated logger methods with new ones

#### What this PR does / why we need it

Fixed: #685 

#### Special notes for your reviewer

nope

### CODE REVIEW

**Comment by miklezzzz** [comment]
@juev sorry to bother you, but we've got some conflicts to resolve :|

**Comment by juev** [comment]
> @juev sorry to bother you, but we've got some conflicts to resolve :|

The rebase is complete. All conflicts have been resolved.

File: pkg/shell-operator/operator.go [hunk 1]
```diff
@@ -631,8 +650,12 @@ func (op *ShellOperator) handleRunHook(t task.Task, taskHook *hook.Hook, hookMet
...
 	}
 
-	if result != nil && result.Usage != nil {
-		taskLogEntry.Debugf("Usage: %+v", result.Usage)
+	if result == nil {
+		return nil
+	}
+
+	if result.Usage != nil {
+		taskLogEntry.Debug("Usage", slog.String("value", fmt.Sprintf("%+v", result.Usage)))
```
Comment (miklezzzz) [inline pkg/shell-operator/operator.go:658]: The same story, result is nil only if some specific error takes place. If we return nil for such a result, it marks the run as successful, which is barely true.
```
} else {
                        success = 1.0
                        taskLogEntry.Infof("Hook executed successfully")
                        res.Status = "Success"
                }
```

Comment (juev) [reply pkg/shell-operator/operator.go]: I just removed this check, at least this is not within the scope of the current task.

**Review by miklezzzz — APPROVED** [review]

**Review by yalosev — APPROVED** [review]

**Review by ldmonster — COMMENTED** [review]
Waiting for log formatting response from observability
1) Log message formatting
2) Log field names (snake case? formatting currently field keys?)

**Review by ldmonster — APPROVED** [review]

**Pull request merged**